src/sdam/srv_polling.ts

```diff
@@ -106,15 +103,15 @@ export class SrvPoller extends TypedEventEmitter<SrvPollerEvents> {
     }
   }
 
+  // TODO(NODE-4817): This method has been left to allow for refactoring with the new logger
   schedule(): void {
     if (this._timeout) {
       clearTimeout(this._timeout);
     }
 
     this._timeout = setTimeout(() => {
-      this._poll().catch(unexpectedRuntimeError => {
-        this.logger.error(`Unexpected ${new MongoRuntimeError(unexpectedRuntimeError).stack}`);
-      });
+      // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
+      this._poll().catch(_unexpectedRuntimeError => {});
```

Comment:
From slack discussion, don't want to remove this as we don't want to have an unhandled promise, so we can just swallow it here.

Reply:
Can you change this to `()=>null` it matches the rest of our catch and release promise handling.

Reply:
we can drop the underscore argument